ground/src/main/java/com/google/android/ground/persistence/remote/firebase/schema/JobNestedObject.kt

```diff
@@ -24,5 +25,5 @@ data class JobNestedObject(
   val defaultStyle: StyleNestedObject? = null,
   val name: String? = null,
   val tasks: Map<String, TaskNestedObject>? = null,
-  val suggestLoiTaskType: String? = null
+  val strategy: String = MIXED.toString()
```

Comment:
I worry that this default value and the default value for the Job model object could get out of sync. I think removing a default from the model might be one way to avoid that.

Reply:
Good idea. Done!